app/react/ToggledFeatures/tocGeneration/specs/actions.spec.ts

```diff
@@ -0,0 +1,99 @@
+import api from 'app/utils/api';
+import backend from 'fetch-mock';
+import * as notificationsTypes from 'app/Notifications/actions/actionTypes';
+import { actions as relationshipActions } from 'app/Relationships';
+import { APIURL } from 'app/config';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import Immutable from 'immutable';
+import { mockID } from 'shared/uniqueID.js';
+import { ClientEntitySchema } from 'app/istore';
+import { tocGenerationActions } from '../actions';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe('reviewToc', () => {
+  it('should store the document with the response of reviewToc', done => {
+    mockID();
+    const fileId = 'fileId';
+
+    backend.post(`${APIURL}files/tocReviewed`, {
+      body: JSON.stringify({ _id: fileId, generatedToc: false }),
+    });
+
+    spyOn(relationshipActions, 'reloadRelationships').and.returnValue({
+      type: 'reloadRelationships',
+    });
+
+    const doc: ClientEntitySchema = {
+      name: 'doc',
+      _id: 'id',
+      sharedId: 'sharedId',
+      defaultDoc: {
+        _id: fileId,
+        generatedToc: true,
+      },
+      documents: [
+        {
+          _id: fileId,
+          generatedToc: true,
+        },
+      ],
+    };
+
+    const updatedEntity = {
+      name: 'doc',
+      _id: 'id',
+      sharedId: 'sharedId',
+      defaultDoc: {
+        _id: fileId,
+        generatedToc: false,
+      },
+      documents: [
+        {
+          _id: fileId,
+          generatedToc: false,
+        },
+      ],
+    };
```

Comment:
Maybe refactor into a quick function that returns either the true or false values and all else being the same instead of this?  Not a really important suggestion, not really needed if time is of the essence.

Reply:
done 👍